lectures/unpleasant.md

```diff
@@ -501,11 +494,13 @@ mystnb:
 plot_path([80, 100], msm)
 ```
 
-Figure  {numref}`fig:unpl1` summarizes outcomes of  two experiments that convey   messages of 
+{numref}`fig:unpl1` summarizes outcomes of  two experiments that convey   messages of 
```

Comment:
please cut "Sargent and Wallace's **unpleasant monetarist arithmetic**" from this sentence.